packages/mobile/src/app/selectors.ts

```diff
@@ -108,4 +108,20 @@ export const linkBankAccountEnabledSelector = (state: RootState) => state.app.li
 
 export const sentryTracesSampleRateSelector = (state: RootState) => state.app.sentryTracesSampleRate
 
+export const supportedBiometryTypeSelector = (state: RootState) => state.app.supportedBiometryType
+
+export const biometryEnabledSelector = (state: RootState) =>
+  state.app.biometryEnabled && !!state.app.supportedBiometryType
+
+export const useBiometrySelector = (state: RootState) => state.app.useBiometry
+
+export const totalRegistrationStepsSelector = createSelector(
+  [choseToRestoreAccountSelector, biometryEnabledSelector],
+  (chooseRestoreAccount, biometryEnabled) => {
+    if (biometryEnabled) {
```

Comment:
This is fine, just sharing how I think it'd be slightly nicer to read imo:
```
let steps = 3
if (chooseRestoreAccount) { 
  steps++ 
}
if (biometryEnabled) { 
  steps++ 
}
return steps
```

this is subjective though, so no need to change if you don't like it

Reply:
i like this, it does show the relationship between the steps more clearly! although i do question if this current relationship will hold true always... i know there is some work to reorganise the onboarding flow, so i think there could be the possibility that we have 2 steps for restore account and 4 steps for new account, for example?

Reply:
Hmm, in that case we'd need to make changes anyways I think? :) Not sure, I'm fine either way

Reply:
yeah you're right. i've updated :)